go/cmd/zk/zkcmd.go

```diff
@@ -510,22 +523,26 @@ func cmdRm(ctx context.Context, subFlags *flag.FlagSet, args []string) error {
 	return nil
 }
 
-func cmdAddAuth(ctx context.Context, subFlags *flag.FlagSet, args []string) error {
-	subFlags.Parse(args)
+func cmdAddAuth(ctx context.Context, subFlags *pflag.FlagSet, args []string) error {
+	if err := subFlags.Parse(args); err != nil {
+		return err
+	}
 	if subFlags.NArg() < 2 {
 		return fmt.Errorf("addAuth: expected args <scheme> <auth>")
 	}
 	scheme, auth := subFlags.Arg(0), subFlags.Arg(1)
 	return zconn.AddAuth(ctx, scheme, []byte(auth))
 }
 
-func cmdCat(ctx context.Context, subFlags *flag.FlagSet, args []string) error {
+func cmdCat(ctx context.Context, subFlags *pflag.FlagSet, args []string) error {
 	var (
 		longListing = subFlags.Bool("l", false, "long listing")
 		force       = subFlags.Bool("f", false, "no warning on nonexistent node")
 		decodeProto = subFlags.Bool("p", false, "decode proto files and display them as text")
```

Comment:
you should move these to `BoolVarP` (and you should also update any other single-character flags to use full names with a `*VarP` function

Reply:
done